### ISSUE TITLE
eom can't build with debian and itstool 2.0.6 (the build is ok with itstool 2.0.5)

### ISSUE THREAD

**Issue by sc0w** [mate-desktop/eom#244]
@monsta @lukefromdc please test the build

the same happens in engrampa, and probably in more repositories.

#### Expected behaviour

The build must be ok

#### Actual behaviour

`/bin/bash: line 3:  6858 Segmentation fault      itstool -m "${mo}" ${d}/C/index.docbook ${d}/C/legal.xml`

The build fails

The issue is confirmed at travis and here locally

NOTE: the build is ok with itstool 2.0.5

I don't understand why is ok in arch and fedora with 2.0.6

see https://travis-ci.org/mate-desktop/eom/jobs/601296862

#### Steps to reproduce the behaviour

build with debian and itstool 2.0.6 (currently in testing and sid)

#### MATE general version

1.22.2

#### Package version

1.22 branch compiled

#### Linux Distribution

debian testing

**Comment by lukefromdc** [comment]
Which Debian version is doing this?  I have Unstable, lack the bandwidth to
downkoad an imstaller for any other version.

Also, with all the changes in mate-common, many older packages don't build
with mate-common from master as of now. I have not had build issues
with any package from master with last Debian Unstable update about
a week agi.

For a test in Unstable, what mate-common version should I use, and I assume the
moat recent EOM from the 1.22 branch is the version to test for build falilures?

**Comment by sc0w** [comment]
in master the build is ok, the issue is in 1.22 branch with debian testing and sid, since the commit https://github.com/mate-desktop/eom/commit/88bb26517e898a5b7e3a4ae8b9d020a488eeeda6 and itstool 2.0.6

I don't know if one more sync with transifex will fix the issue, @raveit65 what do you think?

**Comment by monsta** [comment]
```
/bin/bash: line 3: 19015 Segmentation fault      (core dumped) itstool -m "${mo}" ${d}/C/index.docbook ${d}/C/legal.xml

make[2]: *** [Makefile:603: ru/ru.stamp] Error 139
```
Dammit, Russian translation had two broken strings. I've removed them on Transifex and pulled: https://github.com/mate-desktop/eom/commit/9bb0ed22a82ad859fc5df004fdf8a4bb15c014a0. Fixes build for me, let's also wait for CI to complete.

**Comment by monsta** [comment]
Looks like it wasn't the first itstool segfault:
https://github.com/mate-desktop/atril/issues/407
https://github.com/mate-desktop/mate-applets/issues/388
https://github.com/mate-desktop/mate-user-guide/issues/38

**Comment by lukefromdc** [comment]
Oh yeah: translation related issues have come up from time to time
recently even on master w various packages. Looks like itstool got
a buggy release.

**Comment by sc0w** [comment]
@monsta Thanks, it worked!

btw, we have the same problem with engrampa in 1.22 branch, see:

https://travis-ci.org/mate-desktop/engrampa/jobs/601824543

`/bin/bash: line 3: 12340 Segmentation fault      (core dumped) itstool -m "${mo}" ${d}/C/index.docbook ${d}/C/legal.xml`

@monsta please, can you take a look?

**Comment by monsta** [comment]
Ah, I forgot it was mentioned. Looks similar...
```
make[2]: *** [Makefile:572: ru/ru.stamp] Error 139
```

**Comment by monsta** [comment]
Alright, found and removed it: https://github.com/mate-desktop/engrampa/commit/b01a60a4674adc75ca2288119036c3b64a77487b.

**Comment by sc0w** [comment]
Thanks!

**Comment by monsta** [comment]
But this itstool crash is really ridiculous. Any broken translation and we have a failed build...

**Comment by sc0w** [comment]
> Any broken translation and we have a failed build...

do you mean fedora?

no problem, it will be fixed automatically soon

See https://github.com/mate-desktop/mate-dev-scripts/pull/29#issuecomment-544286128

**Comment by monsta** [comment]
No, I only meant failed builds because of itstool crash.

**Comment by sc0w** [comment]
yes, and I don't understand why arch and fedora didn't crash with same itstool version

**Comment by monsta** [comment]
It seems to be random... in at least one of reports I've mentioned above, there was itstool 2.0.5, which was supposed to be stable. :confused:
